### DISCUSSION TITLE
質問が表示される期間が短い

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#527] [要望・アイデア・不具合]
現在は、一週間程度で質問の一覧から表示されなくなっているようです。
質問のページで最初に表示される「すべて」のカテゴリーの一覧の表示期間が絞られるのは、負荷を下げる為仕方無い気もしますが、フォロー中のタグや、カテゴリーを変更しても一週間分しか表示されないのは不便だと言わざるを得ません。
特に、未解決は単純な放置の場合もありますが、質問の難度の高さ故に未解決という事もあり、時間が経った後に有識者が偶然目にして回答が付く可能性もあります。
改善案としましては、
(1) 「すべて」以外のカテゴリーを選ぶ、もしくはフォロー中のタグを選択した場合は全期間表示する。
(2) 期間を選択可能なボタン等を用意する。（週・月・年・全て　など）
のようなものが考えられますが、いかがでしょう？

**Comment by tanaka-asami415** [comment]
@radian-jp 
ご意見ありがとうございます🙏

ご提案いただいた方法とは違う形になるかもしれませんが、
未解決の質問が多くの方の目に触れるような仕様を検討したいと思います。

ただ、直近の開発優先度などから考えると先になってしまいそうなため
現時点では、1週間より前の質問を探していただく場合は、お手数ですが検索機能を活用いただけると幸いです🙇
https://help.qiita.com/ja/articles/qiita-search-options

**Reply by radian-jp** [reply]
> 現時点では、1週間より前の質問を探していただく場合は、お手数ですが検索機能を活用いただけると幸いです🙇

何らかの情報を探していて記事や過去の質問の回答を検索する、という事はあり得ますが、回答者が質問を検索機能で探して回答するという事は通常無いと考えたほうがよろしいかと思います。
（少なくとも、私は一覧に出てなかったらわざわざ検索してまで回答はしません）

**Comment by kimkim0814** [comment]
@radian-jp
お待たせいたしました!

質問フィードが表示される期間を長くしました。
カテゴリーが「すべて」「未回答」「クローズ」のものは全期間に、
カテゴリーが「回答募集中」のものはパフォーマンスの関係上、1ヶ月間に変更しました。

この度はご意見をいただきありがとうございました。
